Doc/library/smtplib.rst

```diff
@@ -81,9 +83,13 @@ Protocol) and :rfc:`1869` (SMTP Service Extensions).
    An :class:`SMTP_SSL` instance behaves exactly the same as instances of
    :class:`SMTP`. :class:`SMTP_SSL` should be used for situations where SSL is
    required from the beginning of the connection and using :meth:`starttls` is
-   not appropriate. If *host* is not specified, the local host is used. If
-   *port* is zero, the standard SMTP-over-SSL port (465) is used.  The optional
-   arguments *local_hostname*, *timeout* and *source_address* have the same
+   not appropriate. If the optional *host* and *port* parameters are given, the
+   SMTP_SSL :meth:`connect` method is called with those parameters during initialization.
+   If *host* is omitted or an empty string, no connection is made during initialization;
+   you must call :meth:`connect` manually before using the instance. If
+   *port* is zero, the standard SMTP-over-SSL port (465) is used.
```

Comment:
The original of this is quite a bit less accurate.  I would revise it in an identical fashion to the above, again deferring the explanation of what happens with port to the connection docs, including the default port number mention.